runtime/js/99_main.js

```diff
@@ -730,7 +731,7 @@ function removeImportedOps() {
 // FIXME(bartlomieju): temporarily add whole `Deno.core` to
 // `Deno[Deno.internal]` namespace. It should be removed and only necessary
 // methods should be left there.
-ObjectAssign(internals, { core });
+ObjectAssign(internals, { core, nodeGlobals: { ...nodeGlobals } });
```

Comment:
```suggestion
ObjectAssign(internals, { core, nodeGlobals });
```
Might this be fine?

Reply:
No, I don't want to expose the underlying object, because I don't want people to be able to modify it